packages/forms-web-app/src/controllers/examination/process-submission/utils/process.js

```diff
@@ -20,6 +20,8 @@ const handleProcessSubmission = async (session) => {
 			}
 		}
 		logger.info('Processing files complete');
+		examinationSession.submissionId = submissionId;
```

Comment:
should in a setter in the session dir. 

can be in the examination file.

Reply:
Added a setter.